doc/source/user_guide/ug_troubleshooting.rst

```diff
@@ -119,17 +119,24 @@ Several utilities are available that can be helpful when debugging the
 code. Not all of these will work everywhere in the code, due to possible
 conflicts in module dependencies.
 
-*debug\_ice* (**CICE.F90**)
+*debug\_ice* (**ice\_diagnostics.F90**)
```

Comment:
I think this was originally in CICE.F90 due to circular dependencies.  If that's not longer a problem, great!

Reply:
It didn't seem to be a problem to move.